From a965a25d7312b7e0c7ce03986b083971c50f9def Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.xyz>
Date: Wed, 12 Jun 2024 15:21:35 +0800
Subject: [PATCH 123/133] loongarch.cc: rename
 loongarch_memmodel_needs_release_fence =>
 loongarch_memmodel_needs_rel_acq_fence

This commit fixes a previous bad merge.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 gcc/config/loongarch/loongarch.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/loongarch/loongarch.cc b/gcc/config/loongarch/loongarch.cc
index 73d3651f33e..1f6a62ae1c7 100644
--- a/src/gcc/config/loongarch/loongarch.cc
+++ b/src/gcc/config/loongarch/loongarch.cc
@@ -5807,7 +5807,7 @@ loongarch_print_operand (FILE *file, rtx op, int letter)
       break;
 
     case 'G':
-      if (loongarch_memmodel_needs_release_fence ((enum memmodel) INTVAL (op)))
+      if (loongarch_memmodel_needs_rel_acq_fence ((enum memmodel) INTVAL (op)))
 	fputs ("dbar\t0", file);
       break;
 
-- 
2.43.4

